pennylane/devices/tests/test_gates.py

```diff
@@ -386,7 +396,8 @@ def test_basis_state(self, device, basis_state, tol, skip_if):
         """Test basis state initialization."""
         n_wires = 4
         dev = device(n_wires)
-        skip_if(dev, {"returns_probs": False})
+        if isinstance(dev, qml.Device):
```

Comment:
I assume we will clear this (and all subsequent if checks) up with the move away from the older interface?

Reply:
We have a backlogged story to create an alias for `qml.Device` as `qml.devices.LegacyDevice`.